src/components/StoryToShop.js

```diff
@@ -0,0 +1,56 @@
+import styled from 'styled-components'
+
+function StoryToShop() {
+  return (
+    <Wrapper>
+      <Image src="./img/Rettemich.jpg"></Image>
```

Comment:
Is the img src working? I thought you first need to import the img otherwise you can't use it in your component.